### PR TITLE
fix: note added for frontend libraries superblock page

### LINKED ISSUE #50823: Add note on React 18 incompatibility to frontend challenges
### Describe the Issue

Seeing how often this issue still comes up we should probably do something about it.

We have talked about this before but it never really got any traction. The question is where should the note be added and what should it say?

We have an open issue we can link to as well if needed.

https://github.com/freeCodeCamp/freeCodeCamp/issues/45922

### Affected Page

https://www.freecodecamp.org/learn/front-end-development-libraries/

### Your code

```



```

### Expected behavior

Warn the camper about React 18 and the test issues.

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Riya267**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #50823

Project Page Snapshot:-
image

### CODE REVIEW

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -366,7 +366,7 @@
       "In the Front End Development Libraries Certification, you'll learn how to style your site quickly with Bootstrap. You'll also learn how to add logic to your CSS styles and extend them with Sass.",
       "Later, you'll build a shopping cart and other applications to learn how to create powerful Single Page Applications (SPAs) with React and Redux."
     ],
-    "note": "",
+    "note": "Note: React 18 may be incompatible with some front-end challenges.",
```
Comment (naomi-lgbt) [inline client/i18n/locales/english/intro.json:369]:
```suggestion
    "note": "Note: React 18 may be incompatible with the certification projects.",
```

I believe it's the certification projects (with the external test suite) that are the issue.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Comment by lasjorg** [comment]
I might suggest we link to [the issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/45922) as it has an explanation of the problem and gives some solutions for it, such as using the old render method, downgrading to 17, using `flushSync`.

Just saying it may be incompatible with React 18 doesn't seem that helpful.

**Comment by lasjorg** [comment]
I do wonder if having this in the superblock might make some people mistake errors they make in the curriculum challenges for this incompatibility. I have seen people make incorrect assumptions based on less.

Maybe we have to be more explicit about it

> Note: React 18 has known incompatibilities with the tests for the last 5 Front End Development Libraries Projects (see [issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/45922))

**Comment by raisedadead** [comment]
> I might suggest we link to https://github.com/freeCodeCamp/freeCodeCamp/issues/45922 as it has an explanation of the problem and gives some solutions for it, such as using the old render method, downgrading to 17, using flushSync.

Yes - That's a good idea. Although we should use a forum post for such an additional resource.

**Comment by raisedadead** [comment]
Fair points.

> That said, it may be a cleaner and simpler (and even clearer?) approach to add bolded text to the five certification projects? We wouldn't have the cool banner styling, but it would be more direct.

IMHO, the issue is people are busy and don't read. With any number of warnings, we would still get user complaints.

Yup - now that you mention it, adding text instructions would be better than the banner at the superblock level.

@lasjorg, what is your opinion?

**Comment by lasjorg** [comment]
So adding it to the individual challenge instructions instead, like the note at the bottom about Twitter on the [random quote machine](https://www.freecodecamp.org/learn/front-end-development-libraries/front-end-development-libraries-projects/build-a-random-quote-machine) challenge?

Or do we add the warning to the initial text block?

I'm a little worried about a note at the bottom drowning in the challenge instruction wall of text so maybe having it at/near the top would be better?

**Comment by raisedadead** [comment]
Yes, a text note on each of those certifications. The top should be fine.

**Comment by Riya267** [comment]
> Yes, a text note on each of those certifications. The top should be fine.

@raisedadead  is this fine ?
image

File: curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-25-5-clock.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ dashedName: build-a-25--5-clock
 
 # --description--
 
+**Note:** **React 18 has known incompatibilities with the tests for this frontend development libraries Project (see [issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/45922))**
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-25-5-clock.md:11]:
```suggestion
**Note:** **React 18 has known incompatibilities with the tests for this project (see [issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/45922))**
```

Capitalisation, and no need to be so verbose.

File: curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-drum-machine.md [hunk 3]
```diff
@@ -7,6 +7,7 @@ dashedName: build-a-drum-machine
 ---
 
 # --description--
+**Note:** **React 18 has known incompatibilities with the tests for this frontend development libraries Project (see [issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/45922))**
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-drum-machine.md:10]:
```suggestion
**Note:** **React 18 has known incompatibilities with the tests for this project (see [issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/45922))**
```

File: curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-javascript-calculator.md [hunk 4]
```diff
@@ -7,6 +7,7 @@ dashedName: build-a-javascript-calculator
 ---
 
 # --description--
+**Note:** **React 18 has known incompatibilities with the tests for this frontend development libraries Project (see [issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/45922))**
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-javascript-calculator.md:10]:
```suggestion
**Note:** **React 18 has known incompatibilities with the tests for this project (see [issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/45922))**
```

File: curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-markdown-previewer.md [hunk 5]
```diff
@@ -7,6 +7,7 @@ dashedName: build-a-markdown-previewer
 ---
 
 # --description--
+**Note:** **React 18 has known incompatibilities with the tests for this frontend development libraries Project (see [issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/45922))**
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-markdown-previewer.md:10]:
```suggestion
**Note:** **React 18 has known incompatibilities with the tests for this project (see [issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/45922))**
```

File: curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-random-quote-machine.md [hunk 6]
```diff
@@ -7,6 +7,7 @@ dashedName: build-a-random-quote-machine
 ---
 
 # --description--
+**Note:** **React 18 has known incompatibilities with the tests for this frontend development libraries Project (see [issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/45922))**
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-random-quote-machine.md:10]:
```suggestion
**Note:** **React 18 has known incompatibilities with the tests for this project (see [issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/45922))**
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Thanks for all of your work on this. Just a small nitpick.

File: curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-25-5-clock.md (hunk 2 above)
Comment (Riya267) [reply curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-25-5-clock.md]: done

File: curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-drum-machine.md (hunk 3 above)
Comment (Riya267) [reply curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-drum-machine.md]: done

File: curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-markdown-previewer.md (hunk 5 above)
Comment (Riya267) [reply curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-markdown-previewer.md]: done

File: curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-random-quote-machine.md (hunk 6 above)
Comment (Riya267) [reply curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-random-quote-machine.md]: done

**Review by naomi-lgbt — APPROVED** [review]
Awesome! Happy to accept this.

**Review by lasjorg — APPROVED** [review]
LGTM

**Pull request merged**